src/test/java/DetermineAmountOfTimeBeforeOrAfterCourseCompletionTest.java

```diff
@@ -84,7 +132,7 @@ public void shouldCalculateDaysAndHoursBeforeCourseCompletionUsingLocalDateWhenC
         int coursesDuration = 30;
 
         LocalDateTime endDate = DetermineCourseCompletion.getCourseCompletionDate(startDate, coursesDuration);
-        DurationBeforeOrAfterCourseCompletion actual = AmountOfTimeBeforeOrAfterCourseCompletion.calculateAmountOfTimeBeforeOrAfterCourseCompletion(endDate, currentDate, coursesDuration);
+        DurationBeforeOrAfterCourseCompletion actual = AmountOfTimeBeforeOrAfterCourseCompletion.calculateAmountOfTimeBeforeOrAfterCourseCompletion(endDate, currentDate);
         DurationBeforeOrAfterCourseCompletion expected = new DurationBeforeOrAfterCourseCompletion(7, 1);
         assertEquals(expected, actual);
     }
```

Comment:
Where can i find tests from final task example? 
```
Short (Generating report date - 8 June 2020, Monday, 15:00) :
Ivanov Ivan (Java Developer) - Training is not finished. 1 d 3 hours are left until the end.
Sidorov Ivan (J2EE Developer) - Training completed. 3 hours have passed since the end.

```